crates/interpreter/src/gas/calc.rs

```diff
@@ -1,70 +1,10 @@
 use super::constants::*;
-use crate::{num_words, tri, SStoreResult, SelfDestructResult, StateLoad};
-use context_interface::{
-    journaled_state::AccountLoad, transaction::AccessListItemTr as _, Transaction, TransactionType,
-};
+use crate::num_words;
+use context_interface::{transaction::AccessListItemTr as _, Transaction, TransactionType};
 use primitives::{eip7702, hardfork::SpecId, U256};
 
-/// `SSTORE` opcode refund calculation.
-#[allow(clippy::collapsible_else_if)]
 #[inline]
-pub fn sstore_refund(spec_id: SpecId, vals: &SStoreResult) -> i64 {
-    if spec_id.is_enabled_in(SpecId::ISTANBUL) {
-        // EIP-3529: Reduction in refunds
-        let sstore_clears_schedule = if spec_id.is_enabled_in(SpecId::LONDON) {
-            (SSTORE_RESET - COLD_SLOAD_COST + ACCESS_LIST_STORAGE_KEY) as i64
-        } else {
-            REFUND_SSTORE_CLEARS
-        };
-        if vals.is_new_eq_present() {
-            0
-        } else {
```

Comment:
part where we return zero is now here

https://github.com/bluealloy/revm/blob/403f28f96b569d5d25966d43e9b9d1115965a084/crates/interpreter/src/gas/params.rs#L317-L321